src/components/Header.jsx

```diff
@@ -0,0 +1,18 @@
+import styles from '@/styles/Header.module.css';
+
+const Header = () => {
+  const HeaderStyle = {
+    padding: "20px 0",
+        lineHeight: "1.5em",
+        color: "#aeadad",
+        textAlign: "center",
+  }
```

Comment:
@MostafaShahat2021 you tried to add styles as an object here but you also built a CSS file for the `Header`.
Please make your decision to use one of these ways it's necessary for consistency.

Reply:
Hi, @M-AminAlizadeh, sorry for that, I just want to try it, thanks for your advise